fa/fa/convolutional-neural-networks.md

```diff
@@ -0,0 +1,999 @@
+**1. Convolutional Neural Networks cheatsheet**
+
+&#10230;
+<div dir="rtl">
+راهنمای کوتاه شبکه عصبی پیچشی
+</div>  
+
+<br>
+
+
+**2. CS 230 - Deep Learning**
+
+&#10230;
+
+<div dir="rtl">
+یادگیری عمیق - CS 230
+</div>
+<br>
+
+
+**3. [Overview, Architecture structure]**
+
+&#10230;
+
+<div dir="rtl">
+[بررسی اجمالی، معماری ساختار]
+</div>
+<br>
+
+
+**4. [Types of layer, Convolution, Pooling, Fully connected]**
+
+&#10230;
+
+<div dir="rtl">
+[انواع لایه، پیچش، حوضچه، کاملا متصل]
+</div>
+<br>
+
+
+**5. [Filter hyperparameters, Dimensions, Stride, Padding]**
+**5. [Filter hyperparameters, Dimensions, Stride, Padding]** 
+
+&#10230;
+
+<div dir="rtl">
+[فیلتر فرادانسته، ابعاد، گام، لایه گذاری] 
+</div>
+<br>
+
+
+**6. [Tuning hyperparameters, Parameter compatibility, Model complexity, Receptive field]**
+
+ 
+&#10230;
+
+<div dir="rtl">
+[تنظیم فرادانسته‌ها، سازش پذیری پارامتر، پیچیدگی مدل،  زمینه پذیرش]
+</div>
+<br>
+
+
+**7. [Activation functions, Rectified Linear Unit, Softmax]**
+**7. [Activation functions, Rectified Linear Unit, Softmax]** 
+
+&#10230;
+
+<div dir="rtl">
+[توابع فعال ساز، تصحیح واحد خطی، تابع بیشینه هموار] 
+</div>
+<br>
+
+
+**8. [Object detection, Types of models, Detection, Intersection over Union, Non-max suppression, YOLO, R-CNN]**
+
+ 
+&#10230;
+
+<div dir="rtl">
+[تشخیص اشیا، انواع مدل ها، تشخیص، تقاطع بیش از اتحاد، ()محو غیر حداکثری، YOLO ,  شبکه R-CNN]
+</div> 
+<br>
+
+
+**9. [Face verification/recognition, One shot learning, Siamese network, Triplet loss]**
+
+ 
+&#10230;
+
+<div dir="rtl">
+[تایید هویت\تشخیص چهره، یادگیری یکسوی، شبکه سیامس()، خطای سه گانه]  
```

Comment:
Face verification/recognition: تایید\بازشناسایی چهره
I wouldn’t use `تایید هویت` because It's a more general term

<strike>سه گانه</strike>: سه‌گانه

`یکسوی` doesn't seem to be accurate, how about `یک‌باره‌ای`? And I assume it is important to include the original term in parentheses, because often a translation does not convey the shades of meanings contained in the original term.

remove the extra `()`

<strike>سیامس</strike> : Siamese